src/lib/api.js

```diff
@@ -2,7 +2,43 @@ import fetch from "isomorphic-unfetch"
 import queryString from "query-string"
 import { theme } from "./../themes/theme_generator"
 
-export const fetchResultsByQuery = async query => {
+/**
+ * These calls rely on
+ * @param {*} query
+ * @returns
+ */
+export const fetchSiteData = async query => {
+  // TODO no point fetching all of these for all sites if they're not used
+  return await Promise.allSettled([
```

Comment:
I did have `Promise.all` originally but the issue with that is since its the same endpoint (for now!) if one is malformed it wipes out all the rest making scout unusable and that doesn't make sense to do that. 

Since its just (for now!) fetching the filter options it doesn't matter if one errors we'll still have the rest and the user just wont see, for example, the accessibilities section as the promise just returns `[]` for anything that fails and if the whole lot fails it returns `[[],[],[],[]]` - the components are setup to check for empty arrays and just not show.

Reply:
Gotcha! That makes sense, thanks 😄